test/AccountsApiIntegrationTestsWithCallback.js

```diff
@@ -0,0 +1,42 @@
+const docusign = require('../src/index');
+const assert = require('assert');
+const { JWTAuth } = require('./helpers');
+const { apiClient } = require('./constants');
+
+let ACCOUNT_ID = '';
```

Comment:
NIT: Re-assigned variables shouldn't be uppercased https://github.com/airbnb/javascript#naming--uppercase